lib/core/card-view/components/card-view-dateitem/card-view-dateitem.component.html

```diff
@@ -1,5 +1,6 @@
 <div [attr.data-automation-id]="'card-dateitem-label-' + property.key" class="adf-property-label" *ngIf="showProperty() || isEditable()">{{ property.label | translate }}</div>
 <div class="adf-property-value">
+<div class="adf-sr-only" id="{{uniqueId}}">{{'CORE.METADATA.ACCESSIBILITY.DATEPICKER' | translate}}</div>
```

Comment:
can we use  [attr.data-automation-id]="'card-' + property.type + '-value-' + property.key" as uniqueId?

Reply:
this unique id needs to be shared with the aria-describedby shortly below. is it possible to use the same id string in both places?